composer.json

```
@@ -20,6 +22,7 @@
     "require-dev": {
         "phpunit/phpunit": "^9.3",
         "vimeo/psalm": "4.13.1"
```

```suggestion
        "vimeo/psalm": "4.13.1",
        "phpstan/phpstan": "1.9.x-dev"
```

Missing the `phpstan/phpstan` dev dependency used to analyze the code.